### PR TITLE
Fix endpoint

### PR DESCRIPTION
**Pull request by doktordirk**
as descriibed http://aurelia-orm.spoonx.org/decorators.html#endpoint

### CODE REVIEW

File: test/resources/entity/with-endpoint.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import {Entity} from '../../../src/entity';
-import {resource, endpoint} from '../../../src/aurelia-orm';
+import {endpoint} from '../../../src/aurelia-orm';
 
-@resource()
```
Comment (RWOverdijk) [inline test/resources/entity/with-endpoint.js:4]: An entity needs a resource.

Comment (doktordirk) [inline test/resources/entity/with-endpoint.js:4]: that's why i said: http://aurelia-orm.spoonx.org/decorators.html#endpoint
it doesn't there

Comment (RWOverdijk) [inline test/resources/entity/with-endpoint.js:4]: Ah yeah that's wrong.

**Comment by doktordirk** [comment]
alright. but what you thing about the PR. you have quite a lot set to defaults in order to make it simpler for users. removing the need for an empty @resource, could be one one thing in that line

**Comment by RWOverdijk** [comment]
@doktordirk I'm not 100% sure, but I think it then defaults to the class name. I try to discourage that because bundling would break it (bundling renames the class, look at the [gotchas](https://github.com/SpoonX/aurelia-orm#gotchas))

**Comment by doktordirk** [comment]
without have understand/looked into it more, it sounds then that something should be disallowed. having gotchas isn't usually good

**Comment by RWOverdijk** [comment]
Gotchas are fine for rapid prototyping / being unsure about your schema (imo). Or, if you're not planning on minifying

**Comment by doktordirk** [comment]
maybe, but beginners shouldn't run accidentally into them when following basic instructions. dunno if they do, i'll recheck later

**Comment by RWOverdijk** [comment]
That's why there's a small readme.

**Comment by doktordirk** [comment]
replaced by https://github.com/SpoonX/aurelia-orm/issues/89

**Pull request closed without merging**